frontend/src/component/project/Project/ProjectSettings/ProjectSettings.tsx

```diff
@@ -93,7 +93,8 @@ export const ProjectSettings = () => {
             tabs={tabs}
             value={
                 tabs.find(
-                    ({ id }) => id && location.pathname?.includes(`/${id}`),
+                    ({ id }) =>
+                        id && location.pathname?.includes(`/settings/${id}`),
```

Comment:
Maybe ends with would be more accurate?

Reply:
That would be tricky since some of the settings pages have deeper pathing like sub-pages. The more precise solution would be using regex. Think it's worth a try?

Reply:
No, then let's use this, after all including the / in the matching this issue should be solved for good, because we don't allow feature names or projects to have slashes

Reply:
Yes, confirmed: 
![image](https://github.com/Unleash/unleash/assets/455064/f7ae3fcc-4b9f-4c25-ae3a-fd215deb1412)